src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -2280,6 +2280,190 @@ public override IOperation VisitThrow(IThrowOperation operation, int? captureIdF
             }
         }
 
+        public override IOperation VisitUsing(IUsingOperation operation, int? captureIdForResult)
+        {
+            Debug.Assert(operation == _currentStatement);
+
+            SemanticModel semanticModel = ((Operation)operation).SemanticModel;
+            ITypeSymbol iDisposable = semanticModel.Compilation.GetSpecialType(SpecialType.System_IDisposable);
```

Comment:
:memo: I added a work-item to async-streams to come back and revisit for `IAsyncDisposable` case.